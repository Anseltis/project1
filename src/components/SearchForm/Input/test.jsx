import React from 'react'
import renderer from 'react-test-renderer'

import { Input } from './index'

test('renders Input properly', () => {
  const wrapper = renderer.create(<Input />).toJSON()
  expect(wrapper).toMatchSnapshot()
})
