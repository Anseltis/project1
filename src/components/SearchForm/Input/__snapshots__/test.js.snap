// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Input properly 1`] = `
<input
  type="text"
/>
`;
